CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Changelog
 
-## 2.0.5
+## Unreleased
+
+Fix bug in API docs that shows required attributes as optional (#113)
```

Comment:
Code looks good in this commit - but can you fix the URL in the commit message so that it's not split onto multiple lines?

Reply:
Sure I've fixed that.

Reply:
Think we ought to remove this line from this commit now that it's been closed without merging. (Should mean the last commit isn't needed either).